llvm/test/CodeGen/AMDGPU/stack-slot-color-after-sgpr-alloc-equal-size-stack-objects.mir

```diff
@@ -0,0 +1,127 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -verify-machineinstrs -stress-regalloc=3 -start-before=greedy -stop-before=prologepilog -o - %s | FileCheck -check-prefix=SHARE %s
```

Comment:
-mcpu missing. in all new tests.
Since the RUN lines are the same combine all 3 tests into a single .mir file.

Reply:
> -mcpu missing. in all new tests. Since the RUN lines are the same combine all 3 tests into a single .mir file.

Can you enlighten me on what cpu to mention, as I am not sure exactly how to decide on.

Also, kindly comment on the configuration flag related changes.